<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="startPage">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>enter pin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cardNumLineEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menuPage">
     <widget class="QPushButton" name="goToNostaPage">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>nosta rahaa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goToTiliTapahtumatPage">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>100</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>tili tapahtumat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="nostoPage">
     <widget class="QPushButton" name="nosta20">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nosta40">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nosta50">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nosta100">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="saldoLine">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>saldo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testi">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>testi päivitä saldo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuButton1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>takaisin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nosto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tiliTapahtumatPage">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>91</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>tilitapahtumat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuButton2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>takaisin</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
